docarray/math/helper.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional, Tuple
+
+import numpy as np
+
+
+def minmax_normalize(
```

Comment:
couples of questions:

* Is this intended only to work for NumPy array? Why not torch as well? Otherwise, it should be implemented in the computational backend

* If we decide to stick to only NumPy and therefore we don't put it in the computational backend then we should move this to utils.helper or smth like this